srcpkgs/perl-Net-AMQP/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'perl-Net-AMQP'
+pkgname=perl-Net-AMQP
+version=0.06
+revision=1
+archs="x86_64* aarch64* armv6l* armv7l* ppc64le*"
```

Comment:
What's problem with other archs? Should be probably `case $XBPS_TARGET_ARCH; i686*|...) broken="reason";;` if can't be fixed. Please first remove archs= so build fails on CI.

Reply:
It seems that it requires 64bit arch.

```
=> perl-Net-AMQP-0.06_1: running do_check ...
t/01_basic.t .............. ok
t/02_ruby_protocol_doc.t .. ok
t/03_standalone_0_8.t ..... ok
Invalid type 'Q' in pack at /builddir/Net-AMQP-0.06/blib/lib/Net/AMQP/Common.pm line 137.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 2 just after 10.
t/04_table.t .............. 
Dubious, test returned 2 (wstat 512, 0x200)
All 10 subtests passed 
t/50_autodocs.t ........... ok

Test Summary Report
-------------------
t/04_table.t            (Wstat: 512 Tests: 10 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=5, Tests=65,  1 wallclock secs ( 0.04 usr  0.00 sys +  0.94 cusr  0.12 csys =  1.10 CPU)
Result: FAIL
Failed 1/5 test programs. 0/65 subtests failed.
=> ERROR: perl-Net-AMQP-0.06_1: do_check: './Build test' exited with 255
=> ERROR:   in do_check() at common/build-style/perl-ModuleBuild.sh:44
Error: Process completed with exit code 1.
```

Reply:
https://perldoc.perl.org/functions/pack

```
Q  An unsigned quad value.
     (Quads are available only if your system supports 64-bit
      integer values _and_ if Perl has been compiled to support
      those.  Raises an exception otherwise.)
```